src/qibocal/protocols/characterization/classification.py

```diff
@@ -142,208 +237,354 @@ def _acquisition(
+                showlegend=False,
+                mode="markers",
+                marker=dict(size=10, color=get_color_state0(i)),
+            ),
+            row=1,
+            col=i + 1,
+        )
+
+        fig.add_trace(
+            go.Scatter(
+                x=[np.average(state1_data["i"])],
+                y=[np.average(state1_data["q"])],
+                name=f"q{qubit}/{model}: state 1",
+                legendgroup=f"q{qubit}/{model}: state 1",
+                showlegend=False,
+                mode="markers",
+                marker=dict(size=10, color=get_color_state1(i)),
+            ),
+            row=1,
+            col=i + 1,
+        )
+        fig.update_xaxes(
+            title_text=f"i (V)",
+            range=[min_x, max_x],
+            row=1,
+            col=i + 1,
+            autorange=False,
+            rangeslider=dict(visible=False),
+        )
+        fig.update_yaxes(
+            title_text="q (V)",
+            range=[min_y, max_y],
+            scaleanchor="x",
+            scaleratio=1,
+            row=1,
+            col=i + 1,
+        )
+        if len(models_name) != 1:
+            fig_benchmarks.add_trace(
+                go.Scatter(
+                    x=[model],
+                    y=[fit.benchmark_table[qubit]["accuracy"][i]],
+                    mode="markers",
+                    showlegend=False,
+                    marker=dict(size=10, color=get_color_state1(i)),
+                ),
+                row=1,
+                col=1,
+            )
+
+            fig_benchmarks.add_trace(
+                go.Scatter(
+                    x=[model],
+                    y=[fit.benchmark_table[qubit]["training_time"][i]],
+                    mode="markers",
+                    showlegend=False,
+                    marker=dict(size=10, color=get_color_state1(i)),
+                ),
+                row=1,
+                col=2,
+            )
+
+            fig_benchmarks.add_trace(
+                go.Scatter(
+                    x=[model],
+                    y=[fit.benchmark_table[qubit]["testing_time"][i]],
+                    mode="markers",
+                    showlegend=False,
+                    marker=dict(size=10, color=get_color_state1(i)),
+                ),
+                row=1,
+                col=3,
+            )
+
+            fig_benchmarks.update_yaxes(type="log", row=1, col=2)
+            fig_benchmarks.update_yaxes(type="log", row=1, col=3)
+            fig_benchmarks.update_layout(
+                autosize=False,
+                height=COLUMNWIDTH,
+                width=COLUMNWIDTH * 3,
+                title=dict(text="Benchmarks", font=dict(size=TITLE_SIZE)),
+            )
+
+        title_text = ""
+
+        if models_name[i] == "qubit_fit":
+            qubit_model = fit.models[qubit][i]
+            title_text += f"q{qubit}/{model} | average state 0: {np.round(qubit_model.iq_mean0,3)}<br>"
```

Comment:
```suggestion
        title_text = "No fitting data"

        if models_name[i] == "qubit_fit":
            qubit_model = fit.models[qubit][i]
            title_text = f"q{qubit}/{model} | average state 0: {np.round(qubit_model.iq_mean0,3)}<br>"
```
In this way you can avoid to show the empty table for non qubit-fit classifiers.